.github/SUPPORT.md

```diff
@@ -1,13 +1,29 @@
-# Report an issue or request a feature
+# Reporting issues and making requests
 
-You can report issues and request features for this module in the `terraform-ibm-issue-tracker` repo.
+Use GitHub issues in the module repo to report an issue or request a feature. Use the [terraform-ibm-issue-tracker](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new?assignees=&labels=repo-request+%F0%9F%86%95%2Cmodule-onboarding&template=new_repo.yml&title=%5BNew+repo%5D%3A+New+Terraform+module+repo) repo to request a repo for a new module.
 
-Follow these guidelines to help the project maintainers to understand your issue, reproduce the behavior, and find related reports.
+Read our [Code of Conduct](https://github.com/terraform-ibm-modules/.github/blob/main/.github/CODE_OF_CONDUCT.md) to keep our community approachable and respectable. Also, see the [Contributing guide](https://github.com/terraform-ibm-modules/.github/blob/main/.github/CONTRIBUTING.md). For more information about the IBM Cloud Terraform modules project, see the [documentation](https://terraform-ibm-modules.github.io/documentation/).
 
-- Before you create an issue, check the [terraform-ibm-issue-tracker/issues](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues) repo for similar issues.
+## Report an issue
 
-    :information_source: **Tip**: If you find a closed issue that seems similar to what you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+Your participation in identifying and fixing issues help us to improve the modules. Follow these steps to report an issue.
 
-- When you create an issue, feature request, or other idea, use an [issue template](https://github.com/terraform-ibm-modules/terraform-ibm-issue-tracker/issues/new/choose). The information that's requested in the template helps us resolve issues faster.
+1.  Try your issue against the most recent code in the `main` branch in the repository.
```

Comment:
We never want to get people to use main branch. They should always lock into a version. So maybe reword to use latest released version?